docs/wiki/research/PBS/ePBS.md

```diff
@@ -8,221 +8,127 @@
 
 ## TLDR;
 
-Enshrined Proposer-Builder Separation (ePBS) refers to integrating the PBS mechanism directly into the Ethereum blockchain protocol itself, rather than having it operate through external services or addons. This integration aims to formalize and standardize the separation between the roles of block proposers and block builders within the core protocol rules, enhancing the system's efficiency, security, and decentralization.
+Enshrined Proposer-Builder Separation (ePBS) is about integrating the Proposer-Builder Separation (PBS) mechanism directly into Ethereum’s protocol. This formalizes the separation between block proposers and block builders within the core protocol, aiming to improve efficiency, security, and decentralization.
```

Comment:
```suggestion
Large pools have an unfair economic advantage in finding miner extractable value (MEV) due to their ability to choose the contents of the next block. This poses a major threat to Ethereum's consensus network's decentralization. A proposed solution, called proposer-builder separation (PBS), aims to split the block creation process between block proposers and block builders. Currently, PBS is implemented by third-party software like MEV-Boost.

Enshrined Proposer-Builder Separation (ePBS) is a suggested consensus layer upgrade that formalizes the Proposer-Builder Separation (PBS) mechanism directly into Ethereum's protocol. This aims to improve trustlessness, efficiency, and decentralization.
```

Reply:
@thogiti @taxmeifyoucan I expanded tldr to add a bit more context. I have very limited knowledge in this subject; please review if it is accurate.

Reply:
> Large pools have an unfair economic advantage in finding miner extractable value (MEV) due to their ability to choose the contents of the next block. This poses a major threat to Ethereum's consensus network's decentralization.

This is not correct. When Ethereum POS made the choice to outsource block building to 3rd parties (builders) through PBS mechanism, it made a design choice to maximize decentralization by making proposer compute requirements light. Block builders need heavy compute requirements. Over the time, builders specialized and created advanced algorithms and almost dominated the builders market. Today, less than 10% blocks are built by solo builders (not big corporate builder entities). 

ePBS is about bringing back the builder role back into the core protocol.

Reply:
Ahh. That makes sense. I read the post that [proposed PBS](https://ethresear.ch/t/proposer-block-builder-separation-friendly-fee-market-designs/9725) and am not sure of its production usage. 

What you above is useful context. I believe its beneficial to add that to tldr so people have some idea of the background. I'll leave it upto you. The changes overall looks good to me.

Reply:
> Ahh. That makes sense. I read the [proposed PBS](https://ethresear.ch/t/proposer-block-builder-separation-friendly-fee-market-designs/9725) post and am not sure of its production usage.
> 
> What you describe above is useful context. I believe its beneficial to add that to tldr so people have some idea of the background. I'll leave it upto you. The changes overall looks good to me.

The page you linked is a bit old and there were refinements after that. After the merge, builders run MEV-boost under PBS scheme. But it is outside the core protocol. Only the proposer rights is part of the protocol.

The PBS wiki page covers some details on the background and history. https://epf.wiki/#/wiki/research/PBS/pbs 

May be TLDR section is not a place to add the context. We refer to the reader to PBS page in the next paragraph. 

In the wiki page ePBS design specs, we explain the implementation plans of ePBS specs using PTC scheme. This is in active development by Potuz and his team.